index.html

```diff
@@ -0,0 +1,1342 @@
+                            </path>
+                        </g>
+                    </svg>
+                </div>
+                <div class="date-stamp">@ 2021 NEWSWEEK DIGITAL LLC</div>
+            </div>
+
+            <div class="social-icon">
+                <a href="#"><i class="fa fa-facebook-official fa-lg" aria-hidden="true"></i></a>
+                <a href="#"><i class="fa fa-twitter-square fa-lg" aria-hidden="true"></i></a>
+                <a href="#"><i class="fa fa-linkedin fa-lg" aria-hidden="true"></i></a>
+                <a href="#"><i class="fa fa-tumblr fa-lg" aria-hidden="true"></i></a>
+                <a href="#"><i class="fa fa-instagram fa-lg" aria-hidden="true"></i></a>
+            </div>
```

Comment:
- [ ] Fix the size, spacing, and color of the footer links so they look like the original footer:
![Screenshot_2021-01-27 Newsweek(6)](https://user-images.githubusercontent.com/16193212/106056103-0c8c1000-60cd-11eb-955c-c75f27e77709.png)